### PR TITLE
Migrage All Fragments to view binding

### PR DESCRIPTION
**Pull request by HaliksaR**
Please check all functionality!
I dont touch DevSettingsFragment, becouse i dont worked with PreferenceFragmentCompat.

### CODE REVIEW

**Comment by HaliksaR** [comment]
I would also like to introduce a Koin DI into the project. How do you like the proposal?

**Comment by HaliksaR** [comment]
`TooManyFunctions - 11/11 `
Okey( I'll fix it later.

File: app/src/main/java/com/vanced/manager/core/ext/FragmentExt.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.vanced.manager.core.ext
+
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import kotlin.reflect.full.createInstance
+
+fun Fragment.requireSupportFM() = requireActivity().supportFragmentManager
+
+inline fun <reified D : DialogFragment> Fragment.showDialogRefl() {
```
Comment (HaliksaR) [inline app/src/main/java/com/vanced/manager/core/ext/FragmentExt.kt:9]: And check this. Maybe we don't like reflect.

Comment (X1nto) [reply app/src/main/java/com/vanced/manager/core/ext/FragmentExt.kt:9]: showDialog() must be enough, I don't see a reason for keeping showDialogRef() as showDialog() does exactly the same

**Pull request merged**